contracts/Core/AssetManager.sol

```diff
@@ -165,64 +166,37 @@ contract AssetManager is ACL, AssetStorage {
         require(id != 0, "ID cannot be 0");
         require(id <= numAssets, "ID does not exist");
 
-        uint256 epoch = parameters.getEpoch();
-
-        if(jobs[id].assetType == uint256(assetTypes.Job)){
-            jobs[id].active = assetStatus;
-
-            emit JobActivityStatus(
-            id,
-            epoch,
-            jobs[id].active,
-            block.timestamp
-            );
+        if(assetStatus){
+            pendingAssetActivation.push(id);
         }
         else{
-            collections[id].active = assetStatus;
-
-            emit CollectionActivityStatus(
-            id,
-            epoch,
-            collections[id].active,
-            block.timestamp
-            );
-            
+            pendingAssetDeactivation.push(id);
         }
     }
 
     function fulfillAsset(
```

Comment:
please write natspec of what this function does.
Why is it required if we don't allow repeat? then no collection can be fulfilled.
Want to avoid for loops in confirmblock and remove this function if possible.